### PR TITLE
Release Note entry for 2022.09.1

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Adding classic and oie RNs for this weekly release. Includes one amendment to previous monthly release.
- **Is this PR related to a Monolith release?** 2022.09.1

### Resolves:

* [OKTA-527489](https://oktainc.atlassian.net/browse/OKTA-527489)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 1]
```diff
@@ -21,6 +33,7 @@ title: Okta Identity Engine API Products release notes 2022
 | [Clone authentication policies are GA in Preview](#clone-authentication-policies-are-ga-in-preview) | August 31, 2022 |
 | [Improvements to the self-service password reset experience are GA in Preview](#improvements-to-the-self-service-password-reset-experience-are-ga-in-preview) | July 7, 2022 |
 | [Password restriction for shared SWA app accounts is GA in Production](#password-restriction-for-shared-swa-app-accounts-is-ga-in-production) | July 7, 2022 |
+| [Default authenticator enrollment policy settings format](#default-authenticator-enrollment-policy-settings-format) | August 31, 2022 |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:36]:
```suggestion
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 2]
```diff
@@ -68,6 +81,10 @@ Previously, the self-service password reset (SSPR) flow created unnecessary fric
 
 For SWA apps with an account sign-in option set to **Users share a single username and password set by administrator**, only super admins or app admins with permissions for that app can view the password.<!--OKTA-513421-->
 
+#### Default authenticator enrollment policy settings format
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:84]:
```suggestion
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 3]
```diff
@@ -68,6 +81,10 @@ Previously, the self-service password reset (SSPR) flow created unnecessary fric
...
+
+For orgs that recently enabled the Authenticator enrollment policy feature, the new default [authenticator enrollment policy](/docs/reference/api/policy/#authenticator-enrollment-policy ) is created with settings in the authenticator format instead of the factor format. Any existing orgs that have the Authenticator enrollment policy feature enabled, the settings of the default authenticator enrollment policy continues to be in the factor format.<!--OKTA-527304-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:86]:
```suggestion
```

**Review by vanngo-okta — COMMENTED** [review]
Sorry for the confusion Brian, I'm taking out OKTA-537304 because I'm adding it in a separate DOC ACTION PR. Thanks!

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 4]
```diff
@@ -6,6 +6,18 @@ title: Okta Identity Engine API Products release notes 2022
...
+### Weekly release 2022.09.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.09.1](#bugs-fixed-in-2022-09-1)                      | September 14, 2022            |
+
+#### Bugs fixed in 2022.09.1
+
+* The subscription role API didn’t support the API Access Management role. (OKTA-431895)
+* The origin header validation on the `/token` endpoint for cross-origin requests was case-sensitive, which returned an error for redirect URIs using upper-case. (OKTA-516740)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:18]:
```suggestion
* The origin header validation on the `/token` endpoint for cross-origin requests was case-sensitive, which returned an error for redirect URIs using uppercase. (OKTA-516740)
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 5]
```diff
@@ -6,6 +6,18 @@ title: Okta Identity Engine API Products release notes 2022
...
 
+### Weekly release 2022.09.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.09.1](#bugs-fixed-in-2022-09-1)                      | September 14, 2022            |
+
+#### Bugs fixed in 2022.09.1
+
+* The subscription role API didn’t support the API Access Management role. (OKTA-431895)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:17]:
```suggestion
* The Subscription Role API didn’t support the API Access Management role. (OKTA-431895)
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 6]
```diff
@@ -4,6 +4,17 @@ title: Okta API Products release notes 2022
...
 
+### Weekly release 2022.09.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.09.1](#bugs-fixed-in-2022-09-1)                      | September 14, 2022            |
+
+#### Bugs fixed in 2022.09.1
+
+* The subscription role API didn’t support the API Access Management role. (OKTA-431895)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:15]:
```suggestion
* The Subscription Role API didn’t support the API Access Management role. (OKTA-431895)
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 7]
```diff
@@ -4,6 +4,17 @@ title: Okta API Products release notes 2022
...
+* The origin header validation on the `/token` endpoint for cross-origin requests was case-sensitive, which returned an error for redirect URIs using upper-case. (OKTA-516740)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:16]:
```suggestion
* The origin header validation on the `/token` endpoint for cross-origin requests was case-sensitive, which returned an error for redirect URIs using uppercase. (OKTA-516740)
```

**Review by susanharper-okta — APPROVED** [review]
couple nits. approving to not hold you up.

**Comment by brianduffield-okta** [comment]
Merging into release branch.

**Pull request merged**